### DISCUSSION TITLE
User quota visible in webclient

### DISCUSSION THREAD

**Discussion by niele84-dot** [drakkan/sftpgo#1326] [Q&A]
Hi, 
is there any way to have user quota visible in the webclient UI?
The idea is that a user with an assigned space size should be aware of the space left on the server, in order to take actions before it fills up.

Thank you

**Comment by niele84-dot** [comment]
I'm not sure if the change was triggered by my question, but I can see that in sftpgo 2.5.2 the user quota is visible in the webclient.
Thank you @drakkan

**Comment by drakkan** [comment]
> I'm not sure if the change was triggered by my question, but I can see that in sftpgo 2.5.2 the user quota is visible in the webclient. Thank you @drakkan

see https://github.com/drakkan/sftpgo/issues/1327

it is strange that GitHub does not send you a notification when I create an issue from a discussion

**Reply by niele84-dot** [reply]
Indeed, there seems to be no way to enable it from my settings. Anyway I just noticed that on the right hand side there is event section with the link between the issue and the discussion. I will watch that in the future 😅
